pkg/local_workflows/code_workflow/native_workflow.go

```diff
@@ -135,7 +134,7 @@ func sarifLevelToSeverity(level string) string {
 }
 
 // Iterate through the sarif data and create a summary out of it.
-func createCodeSummary(input *sarif.SarifDocument) *json_schemas.TestSummary {
+func CreateCodeSummary(input *sarif.SarifDocument) *json_schemas.TestSummary {
```

Comment:
Suggestion: let's move this to a more common place, now that it is being re-used. similar to the content_types